app/api/email_utils.py

```diff
@@ -74,3 +74,32 @@ def send_email_verification_message(user_name, email):
     html = render_template('email_confirmation.html', confirm_url=confirm_url, user_name=user_name)
     subject = "Mentorship System - Please confirm your email"
     send_email(email, subject, html)
+
+def send_email_request_notification(reciever_id, sender_id, sender_role, end_date, notes):
+    """Sends a mentorship request message to the specified user.
+    1. send_email() for mentorship_relation/send_request route (mentorship_relation.py file):
+    Finds out sender_id, reciever_id and sender_role. It calls send_email_request_notification() 
+    2. This function retrieves usernames and email addresses of sender and reciever. It 
+    converts timestamp to human readable format and sends email.
+    3. email_confirmation.html provides email template
+
+    Args: reciever_id, sender_id, sender_role, end_date, notes
```

Comment:
```suggestion
    Args: receiver_id, sender_id, sender_role, end_date, notes
```

I think you have this type through out the whole code. Can you please fix this?